dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java

```diff
@@ -211,4 +211,13 @@ public static boolean isFeatureFlagOn(final String featureFlagName) {
 		return Config.getBooleanProperty(featureFlagName, true);
 	}
 
+	/**
+	 * Checks the configuration property "DOT_MAIL_FROM_ADDRESS".
+	 * @param fallbackAddress The default email address to use if no global configuration is set.
+	 * @return The global "from" email address if available; otherwise, the fallback address.
+	 */
+	public static String getGlobalFromAddressOrFallback(final String fallbackAddress) {
+		return Config.getStringProperty("DOT_MAIL_FROM_ADDRESS", fallbackAddress);
```

Comment:
remove the `DOT` prefix. It is not required as we always add a `DOT` to the property names when looking for env variables set and docker properties